gradle/mapstruct.gradle

```diff
@@ -1,5 +1,6 @@
-apply plugin: "net.ltgt.apt"
 
 dependencies {
-    apt "org.mapstruct:mapstruct-processor:${mapstruct_version}"
+    compile group: 'org.mapstruct', name: 'mapstruct-jdk8', version: mapstruct_version
+    compileOnly group: 'org.mapstruct', name: 'mapstruct-processor', version: mapstruct_version
+    annotationProcessor group: 'org.mapstruct', name: 'mapstruct-processor', version: mapstruct_version
```

Comment:
Are both `compileOnly` and `annotationProcessor` needed?

Reply:
Good point.Forgot to remove it.